contracts/MicroLotto.sol

```diff
@@ -82,7 +82,9 @@ contract MicroLotto {
                 Ticket storage ticket = wonTickets[i];
                 address ticketAccount = ticket.account;
                 
-                wonPrizes[ticketAccount] = profit;
+                if (wonPrizes[ticketAccount] > 0) wonPrizes[ticketAccount] += profit;
```

Comment:
An uninitialized `wonPrizes` value will have `0` value anyway. No need for if/else.